OpenRA.Mods.Cnc/Graphics/Voxel.cs

```diff
@@ -76,7 +80,7 @@ public float[] TransformationMatrix(uint limb, uint frame)
 			return t;
 		}
 
-		public VoxelRenderData RenderData(uint limb)
+		public ModelRenderData RenderData(uint limb)
```

Comment:
small nit: it is ok, could be changed to match interface signature: RenderData(uint section)
(also for float[] TransformationMatrix(uint section, uint frame))